mappings/net/minecraft/block/Block.mapping

```diff
@@ -18,7 +27,7 @@ CLASS net/minecraft/class_18 net/minecraft/block/Block
 	FIELD field_211 GRASS Lnet/minecraft/class_24;
 	FIELD field_213 BEDROCK Lnet/minecraft/class_18;
 	FIELD field_214 WATER Lnet/minecraft/class_18;
-	FIELD field_216 LAVA Lnet/minecraft/class_18;
+	FIELD field_216 LAVA_MOVING Lnet/minecraft/class_18;
```

Comment:
Why is this `LAVA_MOVING` and `WATER` as well as we have `WATER_STILL` and `LAVA_STILL` so this should have been self-explanatory